api/metrics_service.proto

```diff
@@ -0,0 +1,23 @@
+syntax = "proto3";
+
+package envoy.api.v2;
+import "api/base.proto";
+
+import "google/api/annotations.proto";
+import "metrics.proto";
+
+// Service for streaming metrics to server that consumes the metrics data. It uses Prometheus metric data model as a standard to represent metrics information.
+service MetricsService {
```

Comment:
You will also need to provide a config struct with at least cluster in it. See access log API.